developing-plugins/plugin-developers-guide.md

```diff
@@ -90,22 +103,22 @@ You can test your plugin locally by running GatewayD CLI in development mode. Th
 {: .note }
 > It is recommended to use the `trace` log level to see the logs of your plugin. For more information, see [loggers](/using-gatewayd/global-configuration/loggers).
 
-## Step 5: Test your plugin in the CI pipeline
+## Step 4: Test your plugin in the CI pipeline
 
 Copy the [`test-plugin`](https://github.com/gatewayd-io/gatewayd/blob/213ba09fbf20f0b3923d246d4320dab46fdf8be3/.github/workflows/test.yaml#L61-L144) job of the GatewayD CI pipeline into the `.github/workflows/test.yaml` file. This job will test your plugin using the [GatewayD CLI](/using-gatewayd/CLI) in development mode.
 
-## Step 6: Test your plugin using this `test.yaml` workflow
+## Step 5: Test your plugin using this `test.yaml` workflow
 
 If you have written tests for your plugin, you can use the following workflow to test your plugin. Copy the [`test.yaml`](https://github.com/gatewayd-io/gatewayd-plugin-cache/blob/main/.github/workflows/test.yaml) workflow into the `.github/workflows/` directory of your plugin. This workflow will test your plugin using the [GatewayD CLI](/using-gatewayd/CLI) in development mode.
 
-## Step 7: Publish your plugin to GitHub using this `release.yaml` workflow and this `Makefile`
+## Step 6: Publish your plugin to GitHub using this `release.yaml` workflow and this `Makefile`
 
-If you want to publish your plugin to GitHub, you can use the following workflow and `Makefile` to release your plugin. Copy the [`release.yaml`](https://github.com/gatewayd-io/gatewayd-plugin-cache/blob/main/.github/workflows/release.yaml) workflow and this [`Makefile`](https://github.com/gatewayd-io/gatewayd-plugin-cache/blob/main/Makefile) into the `.github/workflows/` and the root directory of your plugin. This workflow will release your plugin to GitHub.
+If you want to publish your plugin to GitHub, you can use the following workflow and `Makefile` to release your plugin. Copy the [`release.yaml`](https://github.com/gatewayd-io/gatewayd/blob/main/plugin/.template/project/%7B%7B%20plugin_name%20%7D%7D/.github/workflows/release.yaml) workflow and this [`Makefile`](https://github.com/gatewayd-io/gatewayd/blob/main/plugin/.template/project/%7B%7B%20plugin_name%20%7D%7D/Makefile) into the `.github/workflows/` and the root directory of your plugin. This workflow will release your plugin to GitHub.
 
 {: .note }
 > You must modify the example `Makefile`, `release.yaml` and `test.yaml` files to match your plugin.
```

Comment:
Makefile and release.yaml don't need modification.
So these two files can be removed from sentence.